examples/creatives/simid_overlay.js

```diff
@@ -57,4 +58,20 @@ class SimidOverlay extends BaseSimidCreative {
     document.getElementById('log_button').addEventListener(
       'click', sendLogFunction.bind(this));
   }
+
+  sendChangeAdDuration_() {
```

Comment:
This function reads like it will actually send a change Ad Duration call.

Instead what it is really doing is setting up the button listener.

I suggest the name:
sendMessageOnChangeDurationClick_

It's a little verbose but easy to understand.

Reply:
changed name

Reply:
weird, it must have not been added properly. Just committed it again